geopandas_view/view.py

```diff
@@ -341,6 +342,10 @@ def view(
                     "'cmap' is invalid. For categorical plots, pass either valid "
                     "named matplotlib colormap or a list-like of colors."
                 )
+
+        elif callable(cmap):
```

Comment:
Can you add a one-line comment saying that this applied so branca colormap and self-defined function?